lib/engine/game/g_1862/step/acquire.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+require_relative 'merge'
+
+module Engine
+  module Game
+    module G1862
+      module Step
+        class Acquire < G1862::Step::Merge
+          def merge_name
+            'Acquire'
+          end
+
+          def description
+            return 'Choose Survivor' if @merging
+
+            'Acquire Corporation'
+          end
+
+          def process_choose(action)
+            if action.choice == :first
+              @log << "#{@merging.last.name} (non-survivor) will merge into #{@merging.first.name} (survivor)"
```

Comment:
Move log message out of conditional and use survivor and nonsurvivor vars for the message??

Reply:
Should the word merge of acquire be used within the log message?

Reply:
First comment - sure. makes sense
Second comment - I'm hesitant to use the word acquire since acquire works exactly the same as a merge, just at the end of an an turn and not the beginning. There was lots of confusion in the rules forum because people thought "acquiring" implied which company survived.